MP JWT TCK test run fails with Maven 3.9.3 or later

The failure:

```
[ERROR] Suite file .../testsuite/tck/target/tck-suite/suites/tck-base-suite.xml is not a valid file
```

@ayoho Hey Adam, can you verify independently please ? Please run an MP JWT TCK test for OpenLiberty with Maven 3.9.3 or later. I think we may need to fix the way TCK jar is assembled

--- a/tck/src/test/resources/suites/tck-base-suite.xml
+++ b/tck/src/test/resources/suites/tck-base-suite.xml
@@ -1,4 +1,3 @@
-<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 <!--
     Licensed under the Apache License, Version 2.0 (the
     "License"); you may not use this file except in compliance

--- a/tck/src/test/resources/suites/tck-build-suite.xml
+++ b/tck/src/test/resources/suites/tck-build-suite.xml
@@ -1,4 +1,3 @@
-<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 <!--
     Licensed under the Apache License, Version 2.0 (the
     "License"); you may not use this file except in compliance

--- a/tck/src/test/resources/suites/tck-full-suite.xml
+++ b/tck/src/test/resources/suites/tck-full-suite.xml
@@ -1,4 +1,3 @@
-<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 <!--
     Licensed under the Apache License, Version 2.0 (the
     "License"); you may not use this file except in compliance

